theano/tensor/nnet/tests/test_abstract_conv.py

```diff
@@ -687,6 +687,24 @@ def test_bilinear_upsampling_1D(self):
             up_mat_2d = self.get_upsampled_twobytwo_mat(input_x, ratio)
             utt.assert_allclose(f(), up_mat_2d, rtol=1e-06)
 
+    def test_bilinear_upsampling_reshaping(self):
+        """Test bilinear upsampling without giving shape information
```

Comment:
Can you please change the docstring into a comment (i.e. use `#` to comment instead of `"""`)?
We do this only on the unit tests, because that allows the name of the test (e.g. `test_bilinear_upsampling_reshaping`) and its full path to get printed in case of failure.